indico/htdocs/js/indico/Core/Dialogs/Users.js

```diff
@@ -1532,19 +1542,16 @@ type("UserListField", ["IWidget"], {
                 var newUserPopup = new UserDataPopup(
                     $T('New user'),
                     newUser,
-                    function(newData, suicideHook) {
-                        if (newUserPopup.parameterManager.check()) {
-                            newUser.set('isSubmitter', newUser.get('submission'));
-                            self.newProcess([newUser], function(result) {
-                                if (result) {
-                                    self.userList.set(newUserId, newUser);
-                                    self.check(newUser);
-                                    //self._highlightNewUser(newUserId);
-                                    $('.icon-shield[data-id="author_'+newUser.get('email')+'"]').trigger('participantProctChange', [{isSubmitter: newUser.get('isSubmitter') || false}]);
-                                }
-                            });
-                            suicideHook();
-                        }
+                    function(newData) {
+                        newUser.set('isSubmitter', newUser.get('submission'));
+                        self.newProcess([newUser], function(result) {
+                            if (result) {
+                                self.userList.set(newUserId, newUser);
+                                self.check(newUser);
+                                //self._highlightNewUser(newUserId);
+                                $('.icon-shield[data-id="author_'+newUser.get('email')+'"]').trigger('participantProctChange', [{isSubmitter: newUser.get('isSubmitter') || false}]);
```

Comment:
Can we get rid of the typo in `participantProctChange`? Should probably be `participantProtChange`